public/app/screens/main/sections/devices/device/device.jsx

```diff
@@ -0,0 +1,19 @@
+'use strict';
+var React = require('react');
+var StyleMixin = require('mixins/style-mixin');
+
+var Device = React.createClass({
+  mixins: [StyleMixin(require('./style.scss'))],
+  render: function () {
+    return (
+      <div className="cc-devices__device row">
+        <div className="cc-devices__device--title small-6 column">{this.props.device.title}</div>
```

Comment:
This is NOT a part of template scope - it should be addressed when dealing with `device` component